packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -330,6 +330,8 @@
   "Clear_all_unreads_question": "Clear all unreads?",
   "Click_here": "Click here",
   "Click_here_for_more_info": "Click here for more info",
+  "Click_Direct_Message": "Click to Create Direct Message",
+  "Click_Direct_Message_Description": "Skip opening profile tab and goes straight to conversation",
```

Comment:
Either `Skips opening ... and goes ...` or `Skip opening ... and go ...` -- I prefer the former.